packages/@glimmer/component/src/component.ts

```diff
@@ -1,10 +1,10 @@
 import { setComponentManager, setOwner } from '@glimmer/core';
 import GlimmerComponentManager from './component-manager';
-import _GlimmerComponent from '../addon/-private/component';
+import _GlimmerComponent, { Args } from '../addon/-private/component';
 import { DEBUG } from '@glimmer/env';
 
-export default class GlimmerComponent<Args extends {} = {}> extends _GlimmerComponent<Args> {
-  constructor(owner: object, args: Args) {
+export default class GlimmerComponent<S> extends _GlimmerComponent<S> {
```

Comment:
Oops, it's actually this one that needs a default because:

https://github.com/glimmerjs/glimmer.js/blob/ee9fbf09fdf012a512f0ff47872afe3a05a5d51b/packages/%40glimmer/component/index.ts#L1

```suggestion
export default class GlimmerComponent<S = unknown> extends _GlimmerComponent<S> {
```

Reply:
Ahhh, thank you.